script/create-environment-filter.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-param-reassign */
+
+function createEnvironmentFilter(options) {
+  const environmentName = options.buildtype;
+
+  return (files, metalsmith, done) => {
+    for (const fileName of Object.keys(files)) {
+      const file = files[fileName];
+      if (file[environmentName] === false) {
+        delete files[fileName];
+      }
+    }
+    done();
+  };
+}
+
+module.exports = createEnvironmentFilter;
```

Comment:
The environment filter is very small. It just deletes the file from the build if it contains `false` as the build-type's property value.